lte/gateway/python/magma/enodebd/device_config/configuration_init.py

```diff
@@ -86,6 +99,34 @@ def build_desired_config(
     # Attempt to load device configuration from YANG before service mconfig
     enb_config = _get_enb_yang_config(device_config) or \
                  _get_enb_config(mconfig, device_config)
+    # Ho Algorithm
+    _set_algorithm_a1_threshold_rsrp(cfg_desired, enb_config.a1_threshold_rsrp)
+    # _set_algorithm_lte_a1_threshold_rsrq(cfg_desired, enb_config.lte_a1_threshold_rsrq)
+    # _set_algorithm_hysteresis(cfg_desired, enb_config.hysteresis)
+    # _set_algorithm_time_to_trigger(cfg_desired, enb_config.time_to_trigger)
```

Comment:
Why are these lines commented? Should they just be removed? Otherwise add some other comments to explain.

Reply:
The parameters in this part do not provide a path, so they are commented out, like baicells_ rts. The duplicate mapping in py file is the same, and has been deleted at present。